pkg/retriever/retrieve_test.go

```diff
@@ -109,7 +109,9 @@ func TestQueryFiltering(t *testing.T) {
 			cfg := &RetrievalConfig{
 				GetStorageProviderTimeout:   func(peer peer.ID) time.Duration { return time.Second },
 				IsAcceptableStorageProvider: func(peer peer.ID) bool { return true },
-				IsAcceptableQueryResponse:   retriever.isAcceptableQueryResponse,
+				IsAcceptableQueryResponse: func(peer peer.ID, qr *retrievalmarket.QueryResponse) bool {
```

Comment:
why is this change needed?

Reply:
it's not, I had to change it for 2 other tests and obviously did this one mindlessly